alpha-build-machinery/make/targets/golang/verify-update.mk

```diff
@@ -26,6 +28,27 @@ verify-golint:
 	$(GOLINT) $(GO_PACKAGES)
 .PHONY: verify-govet
 
+verify-sec: update-gosec
+ifeq (, $(shell which $(GOSEC) 2>/dev/null))
+	$(GOSEC_GOPATH)/bin/$(GOSEC) \
+		-severity $(GOSEC_SEVERITY) -confidence $(GOSEC_CONFIDENCE) \
+		-exclude $(GOSEC_EXCLUDE) \
+		-quiet $(GO_PACKAGES)
+else
+	$(GOSEC) \
```

Comment:
Wouldn't it be better to always use the same pinned version so people having gosec installed don't get different results then then the rest and CI?

Reply:
That's addressed on the "update-gosec" target

Reply:
addressed in the "update-gosec" comment.